man/fi_efa.7.md

```diff
@@ -103,6 +103,22 @@ provider for AWS Neuron or Habana SynapseAI.
   emulating Read, Write, and Atomic operations (return value is true), or if
   these operations are assisted by hardware support (return value is false).
 
+*FI_OPT_EFA_USE_DEVICE_RDMA - bool*
+: Only available if the application selects a libfabric API version >= 1.18.
```

Comment:
Need to specify that it apply to setopt.

Reply:
it applies to both setopt and getopt.  Should I indicate that explicitly?

Reply:
I see. The commit title says "set use_device_rdma with fi_setopt". I think you just need to apply commit title.

Reply:
commit updated.